nifti2/nifti1.h

```diff
@@ -957,6 +957,21 @@ typedef struct { unsigned char r,g,b; } rgb_byte ;
 
 #define NIFTI_INTENT_SHAPE        2005
 
+ /*! The following intent codes have been used by FSL FNIRT for
+     displacement/coefficient files.  */
```

Comment:
@effigies Discussion that these intent flags support community supported extensions to NIFTI, but implementing the encode/decode behaviors of these intents is not supported by the NIFTI standard.